docs/docs/cloud/how-tos/stream_debug.md

```diff
@@ -0,0 +1,162 @@
+# How to stream debug events
+This guide covers how to stream debug events from your graph (`stream_mode="debug"`).
+
+First let's set up our client and thread:
+
+=== "Python"
+    
+    ```python
+    from langgraph_sdk import get_client
+
+    client = get_client()
+    # create thread
+    thread = await client.threads.create()
+    print(thread)
+    ```
+
+=== "Javascript"
+
+    ```js
+    import { Client } from "@langchain/langgraph-sdk";
+
+    const client = new Client({apiUrl:"whatever-your-deployment-url-is"});
+    # create thread
+    const thread = await client.threads.create();
+    console.log(thread)
+    ```
+
+
+Output:
+
+    {'thread_id': 'd0cbe9ad-f11c-443a-9f6f-dca0ae5a0dd3',
+     'created_at': '2024-06-21T22:10:27.696862+00:00',
+     'updated_at': '2024-06-21T22:10:27.696862+00:00',
+     'metadata': {}}
+
+
+
+Streaming debug events produces responses containing `type` and `timestamp` keys. Debug events correspond to different steps in the graph's execution (e.g. `task`, `task_result`, `checkpoint`).
+
+
+=== "Python"
+
+    ```python
+    # create input
+    input = {
+        "messages": [
+            {
+                "role": "human",
+                "content": "What's the weather in SF?",
+            }
+        ]
+    }
+
+    # stream events
+    async for chunk in client.runs.stream(
+        thread_id=thread["thread_id"],
+        assistant_id="agent",
+        input=input,
+        stream_mode="debug",
+    ):
+        print(f"Receiving new event of type: {chunk.event}...")
+        print(chunk.data)
+        print("\n\n")
+    ```
+
+=== "Javascript"
+
+    ```js
+    # create input
+    var input = {
```

Comment:
(nit) it's more canonical in JS to use `const` instead of `var` unless value actually changes. and for changing values, it's more common to use `let` unless your use case requires to use `var`